examples/threading/threading.c

```diff
@@ -14,6 +14,26 @@ void* threadfunc(void* thread_param)
     // TODO: wait, obtain mutex, wait, release mutex as described by thread_data structure
     // hint: use a cast like the one below to obtain thread arguments from your parameter
     //struct thread_data* thread_func_args = (struct thread_data *) thread_param;
+    
+    struct thread_data *t_data = (struct thread_data *) thread_param;
+    t_data->thread_complete_success = true;
```

Comment:
@jbohn3353 what would happen if another thread read obtained the mutex and read `thread_complete_success` here?  I'd suggest a local variable for the status logic here, and assigning only at the end before mutex unlock.